adlfs/transfer.py

```diff
@@ -217,11 +215,9 @@ class ADLTransferClient(object):
     adlfs.multithread.ADLUploader
     """
 
-    DEFAULT_TMP_PATH = 'tmp'
-
     def __init__(self, adlfs, name, transfer, merge=None, nthreads=None,
-                 chunksize=2**28, blocksize=2**25, tmp_path=DEFAULT_TMP_PATH,
-                 tmp_unique=True, persist_path=None, delimiter=None):
+                 chunksize=2**28, blocksize=2**25, chunked=True,
```

Comment:
I believe all parameters except `unique_temporary`, which is used for testing, are overridden by `ADLDownloader` and `ADLUploader`, so the defaults could be removed.

But, like @begoldsm mentioned, I thought optimal defaults were needed. We allow an advanced user to override these values by creating an `ADLTransferClient` object and passing it into `ADLDownloader` or `ADLUploader`. Removing the defaults would force the user to set each parameter.